bin/generate_dist

```diff
@@ -44,6 +34,17 @@ def generate_dist_files(locale)
     file.write("\n")
   end
 
+  mapping_groups = MappingRule.all.group_by { |record| [record.input_version, record.output_version] }
+  mapping_groups.each do |_, records|
+    directory_path = "dist/integrations/#{records.first.integration.name}"
+    input_version = records.first.input_version.gsub("/", "_")
+    output_version = records.first.output_version.gsub("/", "_")
+    @cli.write_file!("#{directory_path}/#{input_version}_to_#{output_version}.txt") do |file|
+      file.write(MappingRule.as_txt(records, version: @cli.options.version))
+      file.write("\n")
+    end
+  end
```

Comment:
Do we want this in here given this is locale unaware? We're going to overwrites these files `n` times as-is